frontend/src/pages/profilePage.tsx

```diff
@@ -13,19 +14,75 @@ import {
 import { UserContext } from "../contexts/userContext";
 
 import BiFire from "@/assets/bi_fire.png";
-import Frog from "@/assets/frog.png";
+import fire from "@/assets/fire.png";
 import Media from "@/assets/media.png";
+import nature from "@/assets/nature.png";
 import Settings from "@/assets/settings.png";
 import Trophy from "@/assets/trophy.png";
+import water from "@/assets/water.png";
 import ButtonItem from "@/components/ProfileButton";
 import { lightModeColors } from "@/constants/colors";
 
 // import SGDemiBold from "@/assets/fonts/Social-Gothic-DemiBold.otf";
+type Character = {
+  color: string;
+  character: string;
+  characterIcon: ImageSourcePropType;
+};
+
+const characters: Character[] = [
```

Comment:
Nit: can you import `characters` from CharacterCarousel instead of copy/pasting them here?